R/build-home-index.R

```diff
@@ -65,20 +65,11 @@ data_home_sidebar <- function(pkg = ".", call = caller_env()) {
   if (length(html_path)) {
     if (!file.exists(html_path)) {
       rel_html_path <- fs::path_rel(html_path, pkg$src_path)
-
-      msg_fld <- pkgdown_field(
-        pkg, c('home', 'sidebar', 'html'), fmt = TRUE, cfg = TRUE
-      )
-
-      cli::cli_abort(
-        c(
-          "Can't locate {.file {rel_html_path}}.",
-          x = paste0(msg_fld, " is misconfigured.")
-        ),
+      config_abort(
+        pkg,
+        "{.field home.sidebar.html} specifies a file that doesn't exist ({.file {rel_html_path}}).",
```

Comment:
why not 
```suggestion
        "Can't find file {.field home.sidebar.html} ({.file {rel_html_path}}).",
```

Reply:
I like the idea of consistently making the field the first thing in the error message.